src/frontend/src/optimizer/plan_node/logical_agg.rs

```diff
@@ -133,27 +133,31 @@ impl LogicalAgg {
                 TableCatalogBuilder::new(self.ctx().inner().with_options.internal_table_subset());
             let mut column_mapping = vec![];
 
+            let mut upstream_idx_to_state_idx = BTreeMap::new();
```

Comment:
IIUC, `BTreeSet` or `HashSet` is enough?

Reply:
> IIUC, `BTreeSet` or `HashSet` is enough?

sorry, I once thought that we need this column so I use `BTreeMap`, after that I recognized that we need not, but I forget to replace `BTreeMap` with `BTreeSet`. fixed.